frontend/src/static/js/v_authorship.js

```diff
@@ -103,9 +106,35 @@ window.vAuthorship = {
       this.totalLineCount = totalLineCount;
       res.sort((a, b) => b.lineCount - a.lineCount);
 
+      this.fileLineCount = this.convertLineCountToString(this.fileLineCountArray);
       this.files = res;
       this.isLoaded = true;
     },
+
+    addLineCountToFileType(path, lineCount) {
+      const out = {};
+      out.fileType = path.split(".").pop();
```

Comment:
Hmm... however there's also files without an extension / format.
Perhaps we can classify those under `others?`